### ISSUE TITLE
tests3.rs does not check for both tests before continuing

### ISSUE THREAD

**Issue by peterkos** [rust-lang/rustlings#434]
It appears that `tests3.rs` will succeed even if the only defined test is `is_true_when_even`, despite the file comment saying to add a second one to test (presumably) `is_false_when_not_even`:
```
// ... Then write a second test that tests whether we get the result
// we expect to get when we call `is_even(5)`.
```

I feel like it should test for both cases to match the instructions. 
Debug info:
```
❯ rustc --version
rustc 1.44.0 (49cae5576 2020-06-01)
❯ rustlings --version
rustlings 3.0.0
```

**Comment by sautoirs** [comment]
I think `is_false_when_even()` should be renamed `is_false_when_odd()` or `is_false_when_not_even()` as proposed by @peterkos.

**Comment by sautoirs** [comment]
My comment is fixed by #553. Thx
